### DISCUSSION TITLE
Null as result

### DISCUSSION THREAD

**Discussion by JLesuperb** [DaveSkender/Stock.Indicators#1035] [Help and support]
Hi, I'm trying to use the lib but, I'm always having zero as result

StochRsi=>[{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:33:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:34:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:35:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:36:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:37:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:38:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:39:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:40:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:41:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:42:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:43:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:44:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:45:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:46:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:47:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:48:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:49:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:50:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:51:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:52:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:53:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:54:00"},{"StochRsi":null,"Signal":null,"Date":"2023-02-23T10:55:00"}]

**Comment by DaveSkender** [comment]
Many indicators like [Stochastic RSI](https://dotnet.stockindicators.dev/indicators/StochRsi/#response) have an initial incalculable window and possibly early [convergence errors](https://github.com/DaveSkender/Stock.Indicators/discussions/688).  Try with a longer quote history.
